src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -1387,7 +1401,7 @@ public void ReverseStackArgs(ICpu cpu)
         {
             List<object> args = new List<object>();
             object arg = cpu.PopValue();
-            while (arg != null && (!(arg.ToString().Equals(ARG_MARKER_STRING))))
+            while (arg == null || arg.GetType() != OpcodeCall.ARG_MARKER_TYPE)
```

Comment:
This is slightly different logic than before.  That's not a mistake - I realized the logic it had before in regards to null protection was a bit off- it prevented the existence of nulls at all in the arguments by treating them as if they were the arg bottom marker when they aren't.  So that change was deliberate - just in case we start supporting nulls in the future we don't want to mistake nulls for argbottoms.